README.md

```diff
@@ -43,14 +43,20 @@ The plugin uses the native libraries [libphonenumber](https://github.com/google/
 
 | Library        | Version   |
 | -------------- | --------- |
-| libphonenumber | `8.12.52` |
-| PhoneNumberKit | `3.4.4`   |
+| libphonenumber | `8.13.10` |
+| PhoneNumberKit | `3.5.8`   |
 
 ## Usage
 
 ### Setup
 
-No especific setup required: only install the dependency and use :)
+Add the following line to the top of the `Runner` target in your `ios/Podfile`
+```ruby
+target 'Runner' do
+  pod 'PhoneNumberKit/PhoneNumberKitCore', :git => 'https://github.com/marmelroy/PhoneNumberKit', :tag => '3.5.8'
+  ...
+end
+```
```

Comment:
I think that need add this on Podfile example, right?

Reply:
The example Podfile does have the change